static/app/components/feedback/feedbackItem/feedbackItemHeader.tsx

```diff
@@ -26,27 +25,30 @@ const fixIssueLinkSpacing = css`
   }
 `;
 
-export default function FeedbackItemHeader({eventData, feedbackItem}: Props) {
-  return (
-    <VerticalSpacing>
-      <Flex wrap="wrap" flex="1 1 auto" gap={space(1)} justify="space-between">
-        <FeedbackItemUsername feedbackIssue={feedbackItem} detailDisplay />
+type Dimensions = ReturnType<typeof useDimensions>;
+function dimensionsToSize({width}: Dimensions) {
+  if (width < 600) {
+    return 'small';
+  }
+  if (width < 800) {
+    return 'medium';
+  }
```

Comment:
these are kind of random sizes to break on, i'm ok to see it tweaked in prod

Reply:
thats true. i knew something was in there but didn't look. 

I think the breakpoints in here are more related to the width of the column in the page, so that's why they're smaller. it's a good callout, maybe we need sectional breakpoints so we can better handle small screens more often.